### PR TITLE
Added endpoint to get submitted file

### PR DESCRIPTION
**Pull request by BogVin**

### CODE REVIEW

File: app/submissions/views.py
Comment (Chay23) [inline app/submissions/views.py:15]: Actually, I don't believe it won't l be able to handle the traffic load, that's expected.
I suggest to rewrite this part in Java...

File: app/submissions/urls.py
Comment (Chay23) [inline app/submissions/urls.py:9]: It's not acceptable.
Neither in java, not in python.

File: app/core/settings.py
Comment (Chay23) [inline app/core/settings.py:25]: I will not approve it, even if you buy me a beer...

File: app/submissions/views.py
Comment (Chay23) [inline app/submissions/views.py:19]: How long did you think about it?

File: app/assignments/migrations/0001_initial.py
Comment (vladhoi) [inline app/assignments/migrations/0001_initial.py:1]: Please remove from pr all the migrations that don't affect actual tables

File: app/submissions/views.py
Comment (Chay23) [inline app/submissions/views.py:25]: Why did you create file_handle var?

File: app/tests/submissions/test_views.py
Comment (Chay23) [inline app/tests/submissions/test_views.py:79]: No needs of this var

File: app/media/labs/2020/12/17/blob
Comment (vladhoi) [inline app/media/labs/2020/12/17/blob:1]: Please remove all media files from pr, we do not need them on github

**Review by vladhoi — CHANGES_REQUESTED** [review]

File: app/submissions/views.py
Comment (BogVin) [reply app/submissions/views.py:15]: image

**Pull request closed without merging**
